distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Connection.enso

```diff
@@ -147,10 +147,10 @@ type Connection
 
        Arguments:
        - table_name: The name of the table to fetch the column metadata for.
-    # fetch_columns : Text -> Vector [Text, Sql.Sql_Type]
+    # fetch_columns : Text -> Vector [Text, SQL_Type]
     fetch_columns : Text -> Vector Any
```

Comment:
I think maybe we should change the `fetch_columns` implementation:
```diff
...
resolve_column ix =
   ...
- [name, Sql_Type_Data typeid typename]
+ Pair_Data name (Sql_Type_Data typeid typename)
```

then maybe we could do
```suggestion
    fetch_columns : Text -> Vector (Pair Text SQL_Type)
```

Reply:
Fetch Columns will change as part of the read work so not touching for now.